Tests/FluentAssertions.Net45.Specs/Net45.Specs.csproj

```diff
@@ -84,6 +84,9 @@
     <Otherwise />
   </Choose>
   <ItemGroup>
+    <Compile Include="..\FluentAssertions.Net40.Specs\XmlElementAssertionSpecs.cs">
```

Comment:
As far as I could find out, XmlDocument/XmlElement is not supported on portable etc. that is served by the Shared project. Looking at the docs it's only supported in full framework and UWP 10.